python/ray/tune/integration/wandb.py

```diff
@@ -145,11 +146,12 @@ def run(self):
     def _handle_result(self, result):
         config_update = result.get("config", {}).copy()
         log = {}
+        flat_result = flatten_dict(result.copy(), delimiter="/")
```

Comment:
```suggestion
        flat_result = flatten_dict(result, delimiter="/")
```
flatten_dict already does a deepcopy of the input dict